src/fmeval/eval_algorithms/helper_models/helper_model.py

```diff
@@ -140,14 +147,10 @@ def get_helper_scores(self, text_input: List[str]) -> Dict[str, List[float]]:  #
         """
         inputs = self._tokenizer(text_input, return_tensors="pt", truncation=True, padding=True).to(self._model.device)
         scores = torch.sigmoid(self._model(**inputs)[0]).cpu().detach().numpy()
-        results = {}
```

Comment:
Since `text_input` is always a `List` and not a `str`,  we don't need the `isinstance(text_input, str)` section. The fact that it is always a `List` is indicated in the type annotation of the function, and  everywhere that `DetoxifyHelperModel.get_helper_scores` is called, we pass a list: see `DetoxifyHelperModel.__call__` and `evaluate_sample` for `toxicity.py`).

Reply:
Not to mention that if we use the old code, then the output type annotation of this function is wrong; the returned type will be a `Dict[str, float]`. I have already manually verified this.